src/lib.rs

```diff
@@ -1001,4 +1034,27 @@ mod tests {
 
         system.shutdown().unwrap();
     }
+
+    #[test]
+    fn errors() {
+        let mut system = System::new("hi");
+        let full_actor = system.prepare(TestActor).with_capacity(0).spawn().unwrap();
+        let stopped_actor = system.spawn(TestActor).unwrap().recipient();
+
+        let error = full_actor.send(123).unwrap_err();
+        assert_eq!(error.to_string(), "The capacity of TestActor's channel is full.");
+        assert_eq!(
+            format!("{:?}", error),
+            r#"SendError { recipient_name: "TestActor", reason: Full }"#
+        );
+
+        system.shutdown().unwrap();
+
+        let error = stopped_actor.send(456usize).unwrap_err();
```

Comment:
super-nit: type should be inferable?

```suggestion
        let error = stopped_actor.send(456).unwrap_err();
```

Reply:
I see. Thank you for the clarification :)